### PR TITLE
Hotfix/ukus locale issues

### PR DESCRIPTION
**Pull request by prdanelli**
Trying to workout the best way to get around the issues of UK/US spellings in dictionaries and solve issues in how the DOI is prefilling works.

### CODE REVIEW

File: config/authorities/name_type.yml [hunk 1]
```diff
@@ -2,6 +2,6 @@ terms:
 - id: Personal
   term: Personal
   active: true
-- id: Organisational
+- id: Organizational
```
Comment (prdanelli) [inline config/authorities/name_type.yml:5]: All of this code is so this is possible and doesn't break the form toggling and DOI

File: app/services/bolognese/writers/hyku_addons_work_form_fields_writer.rb [hunk 2]
```diff
@@ -409,19 +409,23 @@ def write_involved(type)
 
           meta.dig(type)&.map do |item|
             # transform but don't change original or each time method is run it prepends the key
-            transformed = item.transform_keys { |k| "#{key}_#{k.underscore}" }
+            trans = item.transform_keys { |k| "#{key}_#{k.underscore}" }
```
Comment (okeen) [inline app/services/bolognese/writers/hyku_addons_work_form_fields_writer.rb:412]: Don't you think `transformed` is a better name for this?

File: app/views/records/edit_fields/_creator_organizational.html.erb [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
-<%= render "records/edit_fields/creator_organisational",f: f, type: type, template: template, hash: hash %>
+<div data-toggleable-group="<%= type %>">
+	<% # TODO: workout why setting the name fields as required causes the deposit form to not register the meta data was entered %>
+	<% [
+		{ f: f, field_type: :text, field_slug: :creator_organization_name, field_args: { data: { required: f.object.required?(:creator) } } },
+		{ f: f, field_type: :text, field_slug: :creator_ror },
+		{ f: f, field_type: :text, field_slug: :creator_grid },
+		{ f: f, field_type: :text, field_slug: :creator_wikidata }
+	].each do |field| %>
+		<% options = { field_group: :creator, template: template, hash: hash }.merge(field) %>
```
Comment (okeen) [inline app/views/records/edit_fields/_creator_organizational.html.erb:9]: The code is fine, but I am sure you will love `with_options` https://apidock.com/rails/Object/with_options

**Review by okeen — APPROVED** [review]

File: app/services/bolognese/writers/hyku_addons_work_form_fields_writer.rb (hunk 2 above)
Comment (okeen) [reply app/services/bolognese/writers/hyku_addons_work_form_fields_writer.rb:412]: Sorry, I meant the other way around :D

`transformed` better than `trans`

**Pull request merged**